tests/unit/test_routes.py

```diff
@@ -495,19 +495,33 @@ def test_create_resource(module_client, module_db, fake_auth_from_oc, fake_algol
 
     # Missing Required Fields
     response = client.post('/api/v1/resources',
-                           json=dict(notes="Missing Required fields"),
+                           json=[dict(notes="Missing Required fields")],
                            headers={'x-apikey': apikey})
     assert (response.status_code == 422)
-    assert (isinstance(response.json.get('errors').get(MISSING_PARAMS), dict))
-    assert (isinstance(response.json.get('errors').get(MISSING_PARAMS).get('message'), str))
-    assert ("name" in response.json.get('errors').get(MISSING_PARAMS).get("params"))
-    assert ("name" in response.json.get('errors').get(MISSING_PARAMS).get("message"))
-    assert ("url" in response.json.get('errors').get(MISSING_PARAMS).get("params"))
-    assert ("url" in response.json.get('errors').get(MISSING_PARAMS).get("message"))
-    assert ("category" in response.json.get('errors').get(MISSING_PARAMS).get("params"))
-    assert ("category" in response.json.get('errors').get(MISSING_PARAMS).get("message"))
-    assert ("paid" in response.json.get('errors').get(MISSING_PARAMS).get("params"))
-    assert ("paid" in response.json.get('errors').get(MISSING_PARAMS).get("message"))
+    assert (isinstance(response.json.get('errors')[0].get(MISSING_PARAMS), dict))
+    assert (isinstance(response.json.get('errors')[0].get(MISSING_PARAMS).get('message'), str))
+    assert ("name" in response.json.get('errors')[0].get(MISSING_PARAMS).get("params"))
+    assert ("name" in response.json.get('errors')[0].get(MISSING_PARAMS).get("message"))
+    assert ("url" in response.json.get('errors')[0].get(MISSING_PARAMS).get("params"))
+    assert ("url" in response.json.get('errors')[0].get(MISSING_PARAMS).get("message"))
+    assert ("category" in response.json.get('errors')[0].get(MISSING_PARAMS).get("params"))
+    assert ("category" in response.json.get('errors')[0].get(MISSING_PARAMS).get("message"))
+    assert ("paid" in response.json.get('errors')[0].get(MISSING_PARAMS).get("params"))
+    assert ("paid" in response.json.get('errors')[0].get(MISSING_PARAMS).get("message"))
+
+    # Too many resources in the list
+    response = client.post('/api/v1/resources',
+                            json=[{} for x in range(0,201)],
+                            headers={'x-apikey': apikey})
+
+    assert (response.status_code == 422)
+    assert (isinstance(response.json.get("errors")[0].get("too-long"), dict))
+    assert (isinstance(response.json.get("errors")[0].get("too-long").get("message"), str))
+
+
+    # TODO: Add tests for adding multiple resources.
+    #       - Check that all resources submitted get added
+    #       - Check that error handling works correctly (index is correct for given error object)
```

Comment:
I'm going to add the above tests when I get the chance. Whoever reviews, please feel free to suggest more test cases because I'm not sure what else to check on it. Other than the `Too many resources in the list` test, all of the current tests are just refactored from when we were doing a single resource on this endpoint. The edge cases of multiple resources are not explored yet.